.changeset/calm-dryers-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@skeletonlabs/skeleton": minor
+---
+
+feat: dynamic-animations - Added Transition types, replaced accordion transitions with dynamic transitions.
```

Comment:
Since this will likely drop in isolation - before other components are updated to the standard. I might suggestion:

> feat: Implemented dynamic transitions to the Accordion component.